apps/explorer/src/pages/validator/ValidatorDataTypes.ts

```diff
@@ -142,7 +142,10 @@ export interface NextEpochValidator {
 
 export interface NextEpochValidatorFields {
     consensus_address: number[];
-    name: number[];
+    name: number[] | string;
```

Comment:
I think we can import validator types from the sdk

Reply:
Yeah there's a lot of changes we need to make around this but will punt it until post-testnet